__tests__/aws.cognito.test.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable camelcase */
```

Comment:
I split up the AWS tests into separate files.

Reply:
🍻